authors/apps/articles/models.py

```diff
@@ -5,6 +5,52 @@
 from authors.apps.core.models import TimestampMixin
 
 from cloudinary.models import CloudinaryField
+from django.contrib.contenttypes.models import ContentType
+from django.contrib.contenttypes.fields import (GenericForeignKey,
+                                                GenericRelation)
+from django.db.models import Sum
+
+
+class ReactionManager(models.Manager):
+    """
+    The class enables article likes and dislikes set to be managed separately.
+    """
+    use_related_fields = True
+
+    def likes(self):
+        "fetches reactions with a value greater than zero"
```

Comment:
Ensure your docstring comments have three pairs of double quotation marks in this file.